mindsdb_native/libs/helpers/text_helpers.py

```diff
@@ -10,9 +10,38 @@
 """
 
 from mindsdb_native.libs.constants.mindsdb import *
+from collections import Counter
+import string
 import json
 import hashlib
 import numpy
+import flair
+
+
+def analyze_sentences(data):
+    """
+    :param data: list of str
+    :returns: 
+    tuple(
+        int: nr words total,
+        tuple(
```

Comment:
this is not a tuple (anymore), it's just 3 different parameters, feel free not to leave comments for everything, is a function is small we can read it, incorrect comments do more harm than no comments.